Source/Dafny/Verifier/Translator.cs

```diff
@@ -6260,7 +6260,37 @@ private class WFOptions {
 
       } else if (expr is StmtExpr) {
         var e = (StmtExpr)expr;
+        // If we're inside an "old" expression, then "etran" will know how to translate
```

Comment:
Given the size of `Translator.cs`, and that the code for checking wellformed of expression is about 1200 lines, I suggest we move all of that to `Translator.ExpressionWellformed.cs`. Would you mind doing that in a follow-up PR? I think not getting full Rider IDE support makes it significantly harder to write idiomatic code.

Reply:
Good idea. And I have the same problem with getting Rider's IDE to respond to this long file.

Reply:
I did it in this PR. The code move is a separate commit (and contains only the code move).

Reply:
Even in a separate commit, unless that commit is the first or last in the list, it makes it harder to review the PR.

When trying to comment on part of the diff I also got the message "This diff is outdated, please refresh and try again."

Reply:
I don't understand the diff being outdated. But I can move the new commits to a new PR.